platform/security/servlet/security-servlet-logout/src/main/webapp/less/style.less

```diff
@@ -0,0 +1,74 @@
+table {
+    width: 100%;
+}
+th, td {
+    width: 33%;
+    text-align: center;
+}
+td {
+    padding-top: 10px;
+}
+.is-hidden
+{
+    display: none;
+}
+
+#nav {
+    height: 50px;
+    position: fixed;
+    top: 0px;
+    left: 0px;
+}
+#nav a {
+    margin-left: 10px;
+}
+#nav img {
+    height: 100%;
+}
+#actions {
+    height: ~"calc(100% - 50px)";
```

Comment:
You need a fallback height to use before this one if the browser doesn't support calc